### PR TITLE
Add persistence to notification messages

### PR DESCRIPTION
**Pull request by MissFahrenheit**
If the 'wait' parameter is 0, then never hide the notification message (unless clicked on)

### CODE REVIEW

File: lib/alertify.js [hunk 1]
```diff
@@ -235,7 +235,10 @@
 				var timer = (wait && !isNaN(wait)) ? +wait : this.delay; // Unary Plus: +"2" === 2
 				this.bind(elem, "click", function () {
 					elLog.removeChild(elem);
-				});
+				});				
+				if ( wait == 0 ){
+					return;
+				}
```
Comment (fabien-d) [inline lib/alertify.js]: 1 liner, spacing and strict equal.

``` js
if (wait === 0) return;
```

**Comment by fabien-d** [comment]
I like the feature, but please read the [coding standards](https://github.com/fabien-d/alertify.js/wiki/Coding-Standards) and [Submitting a Pull Request](https://github.com/fabien-d/alertify.js/wiki/Submitting-a-Pull-Request).

Changes should be made to the `src/alertify.js` file.

File: src/alertify.js [hunk 2]
```diff
@@ -224,6 +224,9 @@
 				this.bind(elem, "click", function () {
 					elLog.removeChild(elem);
 				});
+
+				if ( wait === 0 ) return;
```
Comment (fabien-d) [inline src/alertify.js:228]: Still a spacing issue. There should not be spaces between the brackets. And could you also squash your commits into 1? Then I'll merge!

**Comment by catch22** [comment]
+1

**Comment by fabien-d** [comment]
Status update? Looking to merge this in.

**Comment by fabien-d** [comment]
Opened new pull request ^ with same functionality.

**Pull request closed without merging**